likecoin/js/sidebar/src/pages/LikeCoinPluginDocumentSettingPanel.js

```diff
@@ -125,6 +125,20 @@ function LikeCoinPluginDocumentSettingPanel(props) {
                     </button>
                   </div>
                 )}
+                {isCurrentPostPublished && props.ISCNId && (
+                  <button
+                    className='blueBackgroundWhiteTextBtn'
+                    style={{ minWidth: '0', width: '100%' }}
+                    onClick={(e) => {
+                      e.preventDefault();
+                      document
+                        .querySelector('[aria-label="LikeCoin Plugin"]')
+                        .click();
```

Comment:
No other way to trigger the update?

Reply:
Since we always show Update button now, I want to make an extra click to likecoin panel if user is coming from post setting panel. In likecoin panel we have more version details displayed

Reply:
Seems I forgot the push the part for likecoin sidebar sorry :0)